### PR TITLE
Fix Airport communication

### PR DESCRIPTION
**Pull request by JyeSmith**
3 fixes in this PR;

- Add back the deleted OtaPackAirportData() to tx_main.
- Correct the IF statement in rx_main by removing it and relocating OtaPackAirportData().
- Fix the Airport baud if the rx has another protocol select besides CRSF.

### CODE REVIEW

**Comment by JyeSmith** [comment]
Argh, I think I broken linkstats which used to work for Airport.

Ill take another look...

File: src/src/rx_main.cpp [hunk 1]
```diff
@@ -495,10 +495,7 @@ bool ICACHE_RAM_ATTR HandleSendTelemetryResponse()
     alreadyTLMresp = true;
     otaPkt.std.type = PACKET_TYPE_TLM;
 
-    bool noAirportDataQueued = firmwareOptions.is_airport && apOutputBuffer.size() == 0;
-    bool noTlmQueued = !TelemetrySender.IsActive() && noAirportDataQueued;
-
-    if (NextTelemetryType == ELRS_TELEMETRY_TYPE_LINK || noTlmQueued)
+    if (NextTelemetryType == ELRS_TELEMETRY_TYPE_LINK)
```
Comment (wvarty) [inline src/src/rx_main.cpp:498]: This looks like a regression to me.
The previous code sent link stats packets if there is no telem data queued.
This new change will only send a link stats pkt every 2nd telem slot if no data is queued.

Comment (JyeSmith) [reply src/src/rx_main.cpp]: Nice find on catching the incorrect AP buffer!

**Comment by JyeSmith** [comment]
Latest commit from Wez looks good.

**Review by wvarty — APPROVED** [review]

**Pull request merged**